modules/aurora-mysql-resources/modules/mysql-user/main.tf

```diff
@@ -91,7 +91,6 @@ resource "mysql_user" "default" {
   user               = local.db_user
   host               = "%"
   plaintext_password = local.db_password
-  #  depends_on         = [var.instance_ids]
```

Comment:
Are you sure this is not needed? What is preventing Terraform from trying to create the user before the database is created?

Reply:
this was originally commented out when starting the update, so I will uncomment and return this check